director/settings.py

```diff
@@ -44,6 +44,9 @@ def __init__(self, home_path=None, config_path=None):
         self.FLOWER_URL = env.str("DIRECTOR_FLOWER_URL", "http://127.0.0.1:5555")
         self.WORKFLOWS_PER_PAGE = env.int("DIRECTOR_WORKFLOWS_PER_PAGE", 1000)
         self.REFRESH_INTERVAL = env.int("DIRECTOR_REFRESH_INTERVAL", 30000)
+        self.REPO_LINK = env.str(
+            "DIRECTOR_REPO_LINK", "https://github.com/ovh/celery-director"
+        )
```

Comment:
After discussing with the team we don't understand why changing the repository link can be useful for the end users. Can you explain us a real use case please?

If not possible please remove it completely ;)

Reply:
With user stories :)

- As celery director end-user, I want to view/edit workflow definitions to make change/review in worklfows.

- As celery director advanced user, I want to view task definitions to debug failed workflows


It will be the same feature as provided by [mkdocs material](https://squidfunk.github.io/mkdocs-material/setup/adding-a-git-repository/#repository).

If those needs make sense to you, we can still find a way to keep the official repo link in the main UI.

Reply:
Ok it's clear for me, thanks for the clarification :)